### PR TITLE
Create feynman-PBS.yaml

### PR DESCRIPTION
**Pull request by biozit**
adding new cluster

### CODE REVIEW

File: topology/University of Campinas/CCJDR/feynman-PBS.yaml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+# SupportCenter is one of the support centers in topology/support-centers.yaml
+SupportCenter: 0
+
+# GroupDescription is a long description of the resource group; may be multiple lines.
+GroupDescription: This is a cluster of HPC used for the Dune
+
+# If you have an up-to-date local git clone, fill GroupID with the output from `bin/next_resource_group_id`
+# Otherwise, leave it blank and we will fill in the appropriate value for you.
+GroupID:
```
Comment (brianhlin) [inline topology/University of Campinas/CCJDR/feynman-PBS.yaml:12]:
```suggestion
GroupID: 1108
```

File: topology/University of Campinas/CCJDR/feynman-PBS.yaml [hunk 2]
```diff
@@ -0,0 +1,119 @@
+# Production is true if the resource is for production and not testing use
+Production: false
+
+# SupportCenter is one of the support centers in topology/support-centers.yaml
+SupportCenter: 0
```
Comment (brianhlin) [inline topology/University of Campinas/CCJDR/feynman-PBS.yaml:5]:
```suggestion
SupportCenter: Self Suppported
```

File: topology/University of Campinas/CCJDR/feynman-PBS.yaml [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  # Resource Names need to be unique across all resources in the OSG.
+  feynman:
+    # Active is true if the resource is accepting requests, and false otherwise.
+    # When first registering a resource, set this to false. Set it to true when it's ready for production.
+    Active: false
+    # Description is a long description of the resource; may be multiple lines
+    Description: This is a HPC CLuster for CCJDR to used by Dune
+    # If you have an up-to-date local git clone, fill ID with the output from `bin/next_resource_id`
+    # Otherwise, leave it blank and we will fill in the appropriate value for you.
+    ID:
```
Comment (brianhlin) [inline topology/University of Campinas/CCJDR/feynman-PBS.yaml:28]:
```suggestion
    ID: 1126
```

File: topology/University of Campinas/CCJDR/feynman-PBS.yaml [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+    #   - <HOSTNAME2>
+
+    ### DN (optional except for XCache resources) is the DN of the host cert of the resource
+    # in OpenSSL 1.0 format (i.e. /DC=org/DC=incommon/C=US/...)
+    # DN: <DN>
+
+    # Services is one or more services provided by this resource;
+    # valid services are listed in topology/services.yaml with the format "<SERVICE NAME>: <ID>"
+    Services:
+      <SERVICE NAME>:
```
Comment (brianhlin) [inline topology/University of Campinas/CCJDR/feynman-PBS.yaml:72]:
```suggestion
      CE:
```

From the FQDN, I gather that this is an HTCondor-CE?

**Review by brianhlin — COMMENTED** [review]
Hi @biozit, thanks for the registration. I've made some GitHub suggestions that you should be able to merge through the GitHub UI as we all as a question about the service on this host

File: topology/University of Campinas/CCJDR/feynman-PBS.yaml (hunk 4 above)
Comment (biozit) [reply topology/University of Campinas/CCJDR/feynman-PBS.yaml:72]: condor.ifi.unicamp.br

Comment (brianhlin) [reply topology/University of Campinas/CCJDR/feynman-PBS.yaml]: Thanks for the response @biozit. What grid services do you have installed on `condor.ifi.unicamp.br`? Is it an HTCondor-CE?

Comment (biozit) [reply topology/University of Campinas/CCJDR/feynman-PBS.yaml]: The HTCondor-CE is installed on condor.ifi.unicamp.br, however, the jobs will be routed to another host,

File: topology/University of Campinas/CCJDR/feynman-PBS.yaml [hunk 5]
```diff
@@ -0,0 +1,119 @@
+# Production is true if the resource is for production and not testing use
+Production: false
+
+# SupportCenter is one of the support centers in topology/support-centers.yaml
+SupportCenter: Self Suppported
```
Comment (brianhlin) [inline topology/University of Campinas/CCJDR/feynman-PBS.yaml:5]:
```suggestion
SupportCenter: Self Supported
```

Fixing a typo

**Comment by biozit** [comment]
Please, there is anything else to fix?

File: topology/University of Campinas/CCJDR/feynman-PBS.yaml [hunk 6]
```diff
@@ -0,0 +1,119 @@
+# Production is true if the resource is for production and not testing use
+Production: false
```
Comment (brianhlin) [inline topology/University of Campinas/CCJDR/feynman-PBS.yaml:2]: Is this registration for a test CE?

File: topology/University of Campinas/CCJDR/feynman-PBS.yaml [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+# Otherwise, leave it blank and we will fill in the appropriate value for you.
+GroupID: 1108
+
+# Resources contains one or more resources in this
+# ResourceGroup. A resource provides one or more services
+Resources:
+  # Resource Name should be a short descriptor of the resource.
+  # e.g. the Center for High Throughput Computing's GlideinWMS Frontend is "CHTC-glidein2"
+  # Resource Names need to be unique across all resources in the OSG.
+  feynman:
```
Comment (brianhlin) [inline topology/University of Campinas/CCJDR/feynman-PBS.yaml:20]:
```suggestion
  CCJDR-feynman:
```

Resource names need to be unique across the OSG so let's prefix it with your site name

File: topology/University of Campinas/CCJDR/feynman-PBS.yaml [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+    # Active is true if the resource is accepting requests, and false otherwise.
+    # When first registering a resource, set this to false. Set it to true when it's ready for production.
+    Active: false
```
Comment (brianhlin) [inline topology/University of Campinas/CCJDR/feynman-PBS.yaml:23]: This setting is fine for now but once your CE is ready to accept pilots, you'll want to set this to `Active: True`

**Review by brianhlin — COMMENTED** [review]
A few more things but the registration is looking mostly good

File: topology/University of Campinas/CCJDR/feynman-PBS.yaml (hunk 8 above)
Comment (biozit) [reply topology/University of Campinas/CCJDR/feynman-PBS.yaml:23]: please, could you set this to true?

Comment (brianhlin) [inline topology/University of Campinas/CCJDR/feynman-PBS.yaml:23]:
```suggestion
    Active: true
```

File: topology/University of Campinas/CCJDR/feynman-PBS.yaml (hunk 6 above)
Comment (brianhlin) [reply topology/University of Campinas/CCJDR/feynman-PBS.yaml:2]: @biozit is this for a test or production CE?

Comment (biozit) [reply topology/University of Campinas/CCJDR/feynman-PBS.yaml:2]: yes, it is.

**Review by brianhlin — APPROVED** [review]
LGTM

**Pull request merged**